### PR TITLE
Fix the kebab-case for css properties warning

### PR DESCRIPTION
**Pull request by sagnik3788**
### What change does this PR introduce?
solve the kebab-case for css properties warning

### Why was this change needed?
solve issue #4294 @LetItRock

### CODE REVIEW

File: apps/web/src/design-system/button/Button.styles.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const getSubtleStyles = (theme) => {
           backgroundColor: dark ? theme.white : 'transparent',
           backgroundClip: 'none !important',
           color: 'transparent',
-          '-webkit-text-fill-color': 'initial !important',
+          WebkitTextFill: 'initial !important',
```
Comment (scopsy) [inline apps/web/src/design-system/button/Button.styles.ts:69]: @LetItRock I think it should be: `WebkitTextFillColor` ?

Comment (sagnik3788) [reply apps/web/src/design-system/button/Button.styles.ts:69]: is there anything i change @scopsy @LetItRock

Comment (davidsoderberg) [reply apps/web/src/design-system/button/Button.styles.ts:69]: Hi @sagnik3788 if the error that is shown in the issue is not showing anymore I see this as the solution to the issue and nothing else needed to change 😄

**Comment by sagnik3788** [comment]
if everything is okay can you merge my pr @davidsoderberg

**Comment by davidsoderberg** [comment]
Hi @sagnik3788, I think it should be `WebkitTextFillColor` or have you checked so the css is applied with your solution?

**Comment by sagnik3788** [comment]
Yes, you are correct  it should be WebkitTextFillColor due to the camelCase property naming convention . i correct this

File: apps/web/src/design-system/button/Button.styles.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const getSubtleStyles = (theme) => {
           backgroundColor: dark ? theme.white : 'transparent',
           backgroundClip: 'none !important',
           color: 'transparent',
-          '-webkit-text-fill-color': 'initial !important',
+        WebkitTextFillColor: 'initial !important',
```
Comment (ainouzgali) [inline apps/web/src/design-system/button/Button.styles.ts:69]: This line should have the same indentation as the others.

**Review by ainouzgali — COMMENTED** [review]
@sagnik3788 please run the prettier and I'll approve this. Thanks!

**Review by LetItRock — APPROVED** [review]

**Pull request merged**